### DISCUSSION TITLE
exclude all images and payloads? Just have actual pages

### DISCUSSION THREAD

**Discussion by pixelpaulaus** [nuxt-modules/algolia#210] [Q&A]
I noticed it adds everything to the index. Images, payloads and more.
How can I exclude all the extra stuff and only have my actual pages included?

**Comment by Baroshem** [comment]
Hey @pixelpaulaus 

Could you share more details? I don't understand what component/composable you are using or if it is even related to the module or the Algolia itself.

If you have a reproduction repository, that would also be useful :)
